docs/es/docs/deployment/manually.md

```diff
@@ -65,10 +65,10 @@ And run your application the same way you have done in the tutorials, but withou
 
     </div>
 
-You might want to set up some tooling to make sure it is restarted automatically if it stops.
+Tal vez quedrás configurar algunas herramientas, para asegurarte que se reinicie automáticamente si llega a parar.
 
-You might also want to install <a href="https://gunicorn.org/" class="external-link" target="_blank">Gunicorn</a> and <a href="https://www.uvicorn.org/#running-with-gunicorn" class="external-link" target="_blank">use it as a manager for Uvicorn</a>, or use Hypercorn with multiple workers.
+También quedrás instalar <a href="https://gunicorn.org/" class="external-link" target="_blank">Gunicorn</a> y <a href="https://www.uvicorn.org/#running-with-gunicorn" class="external-link" target="_blank">usarlo como administrador de Uvicorn</a>, o usar Hypercorn con multiples workers.
```

Comment:
I think `quedrás` is not a valid Spanish word. I would use instead `desee` or `quieras`.